### PR TITLE
Allow $refs in the EncodingObject#headers map

### PR DESCRIPTION
**Pull request by tfesenko**
As discussed in PR #1126

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -1553,7 +1553,7 @@ A single encoding definition applied to a single schema property.
...
-<a name="encodingHeaders"></a>headers | Map[`string`, [Header Object](#headerObject)] | A string map allowing additional information to be provided as headers, for example `Content-Disposition`.  `Content-Type` is described separately and SHALL be ignored in this section. This property SHALL be ignored if the request body media type is not a `multipart`.
+<a name="encodingHeaders"></a>headers | Map[`string`, [Header Object](#headerObject) \| [Reference Object](#referenceObject)] | A string map allowing additional information to be provided as headers, for example `Content-Disposition`.  `Content-Type` is described separately and SHALL be ignored in this section. This property SHALL be ignored if the request body media type is not a `multipart`.
```
Comment (DavidBiesack) [inline versions/3.0.md]: Not sure if "A string map" is accurate anymore. It's a map of String -> [Header Object | $ref ]

Comment (tfesenko) [inline versions/3.0.md]: A good point, @DavidBiesack !
So, the current description is
> A string map allowing additional information to be provided as headers, <...>

Would it be better if I remove the "string" in "a string map":
>  A map allowing additional information to be provided as headers, <...>

Comment (DavidBiesack) [inline versions/3.0.md]: yes, just say "a map ..."

Comment (tfesenko) [inline versions/3.0.md]: 'a string map' -> 'a map' - done in de6ec79

**Pull request merged**